index.js

```diff
@@ -173,3 +173,5 @@ module.exports = fp(fastifyCors, {
   fastify: '3.x',
   name: 'fastify-cors'
 })
+
+module.exports.vary = vary
```

Comment:
Can you move `vary` to its own file so you can test it separately without exposing it to the public?

Reply:
Is there a reason why we can't use the `vary` function of the `vary` module?

Reply:
that works with node core `ServerResponse` object.

Reply:
ok, thanks

Reply:
Do you think I should add more tests for the `vary` function itself?